src/Symfony/Component/PropertyInfo/Extractor/PhpDocExtractor.php

```diff
@@ -217,9 +225,11 @@ private function filterDocBlockParams(DocBlock $docBlock, string $allowedParam):
     }
 
     /**
+     * Gets a property DocBlock.
```

Comment:
It's a little bit confusing, that this and not the `get*` method _gets_ the docblock

You know what I mean? Maybe remove the description entirely?

Reply:
Removed